tapestry.js

```diff
@@ -1021,36 +1049,37 @@ function getNodesDimensions(dataset) {
     };
 }
 
+// get tapestry SVG's height and width
 function getTapestryDimensions() {
 
     var nodeDimensions = getNodesDimensions(dataset);
-    var tapestryWidth = nodeDimensions['x'];
-    var tapestryHeight = nodeDimensions['y'];
+    var tapestryWidth = nodeDimensions.x;
+    var tapestryHeight = nodeDimensions.y;
 
-    var tapestryAspectRatio = nodeDimensions['x'] / nodeDimensions['y'];
+    var tapestryAspectRatio = nodeDimensions.x / nodeDimensions.y;
     var tapestryBrowserRatio = tapestryWidth / getBrowserWidth();
 
-    if (tapestryHeight > getBrowserHeight() && tapestryAspectRatio < 1) {
+    if (tapestryHeight > getBrowserHeight() && tapestryAspectRatio < 1) {  // this was originally 1 instead of 5 - ford
```

Comment:
It's not clear how noting the original value here is helpful. Could you please remove this comment?